keps/sig-node/3066-subsecond-probes/kep.yaml

```diff
@@ -0,0 +1,30 @@
+title: Subsecond Probes
+kep-number: 3066
+authors:
+  - "@mikebrow"
+  - "@psschwei"
+  - "@MHBauer"
+owning-sig: sig-node
+participating-sigs:
+  - sig-node
```

Comment:
please add sig-network as participating sig to validate the potential scalability issues